documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Applications (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Application (OVA) file migrations:
+
+* All OVA files are created by or compatible with VMware vSphere.
+* The OVA files are in a shared NFS share that has the following security settings: `rw,insecure`.
+* Each OVA file is in a separate folder in the NFS share -  sub-folders are not checked.
```

Comment:
Also, it's OK for all the OVA files to be in the same folder under the share, no need for separated folders.

Reply:
@RichardHoch same here